librz/asm/p/asm_arm_cs.c

```diff
@@ -294,10 +310,10 @@ RzAsmPlugin rz_asm_plugin_arm_cs = {
 	.bits = 16 | 32 | 64,
 	.endian = RZ_SYS_ENDIAN_LITTLE | RZ_SYS_ENDIAN_BIG,
 	.disassemble = &disassemble,
-	.mnemonics = mnemonics,
+	//.mnemonics = mnemonics,
```

Comment:
Just remove this?

Reply:
i thought i have reintroduced it. i must have forgotten to cherrypick that change.